packages/fxa-auth-server/test/remote/password_change_jwt_tests.js

```diff
@@ -0,0 +1,195 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const { assert } = require('chai');
+const Client = require('../client')();
+const config = require('../../config').default.getProperties();
+const TestServer = require('../test_server');
+const jwt = require('jsonwebtoken');
+const uuid = require('uuid');
+
+describe('#integration - remote password change JWT', function () {
```

Comment:
Nice! Ping nick about these changes. He side stepped remote tests in his PR cause he was getting a little bogged down in some of the setup.